src/components/navigation-bar/navigation-bar.pw.tsx

```diff
@@ -0,0 +1,315 @@
+    test(`should render with orientation set to bottom and offset prop set to ${offset} px`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NavigationBarComponent
+          offset={`${offset}px`}
+          orientation="bottom"
+          position="fixed"
+        />
+      );
+
+      await expect(navigationBar(page)).toHaveCSS("bottom", `${offset}px`);
+    });
+  });
+
+  (["top", "bottom"] as const).forEach((orientation) => {
+    test(`should render NavigationBar component with orientation prop set to ${orientation}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NavigationBarComponent orientation={orientation} position="fixed" />
+      );
+
+      await expect(navigationBar(page)).toHaveCSS(orientation, "0px");
+    });
+  });
+
+  (["fixed", "sticky"] as NavigationBarProps["position"][]).forEach(
+    (position) => {
+      test(`should render with position prop set to ${position} and work as expected`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <div
+            style={{
+              height: "250px",
+            }}
+          >
+            <NavigationBar
+              position={position}
+              orientation="top"
+              offset="25px"
+              aria-label="header"
+            >
+              <Box display="flex" flex="1" maxWidth="1000px" margin="0 auto">
+                <Menu display="flex" flex="1">
+                  <MenuItem flex="1" onClick={() => {}}>
+                    Menu Item One
+                  </MenuItem>
+                </Menu>
+              </Box>
+            </NavigationBar>
+            <div
+              style={{
+                height: "1000px",
+              }}
+            />
+            <NavigationBar
+              position={position}
+              orientation="bottom"
+              offset="25px"
+              aria-label="footer"
+            >
+              <Box display="flex" flex="1" maxWidth="1000px" margin="0 auto">
+                <Menu display="flex" flex="1">
+                  <MenuItem flex="1" onClick={() => {}}>
+                    Menu Item One
+                  </MenuItem>
+                </Menu>
+              </Box>
+            </NavigationBar>
+          </div>
+        );
+
+        await expect(navigationBar(page).nth(0)).toBeVisible();
+
+        await expect(navigationBar(page).nth(1)).toBeVisible();
+
+        await page.evaluate(() => window.scrollTo(1000, 1000));
+
+        if (position === "fixed") {
+          await expect(navigationBar(page).nth(0)).toBeInViewport();
+        } else {
+          await expect(navigationBar(page).nth(0)).not.toBeInViewport();
+        }
+
+        await expect(navigationBar(page).nth(1)).toBeVisible();
+      });
+    }
+  );
+
+  test.describe("Accessibility tests for NavigationBar component", () => {
+    test("should pass accessibility tests Default story", async ({
```

Comment:
**nitpick/suggestion(non-blocking):** Sorry, I should have mentioned this yesterday when I suggested moving them to the `test-pw` file. As these are no longer stories it should just be:

```suggestion
    test("should pass accessibility tests Default example", async ({
```

Same for the others below too. Sorry! My bad 😢